packages/gestalt-datepicker/src/DatePicker.js

```diff
@@ -212,7 +212,7 @@ DatePickerWithForwardRef.propTypes = {
   nextRef: PropTypes.oneOfType([
     PropTypes.func,
     PropTypes.shape({
-      current: PropTypes.instanceOf(Node),
+      current: PropTypes.node,
```

Comment:
The reference cannot be any React node, but has to be an HTMLDivElement, right? Should this be `PropTypes.elementType`?

Reply:
Going to r+ to unblock the Gestalt release (once we SSR our docs we will be removing propTypes anyhow)

Reply:
Hey @jackhsu978, we can probably be more specific with `PropTypes.elementType`, `PropTypes.node` is more general but did the job of fixing the bug. https://reactjs.org/docs/typechecking-with-proptypes.html
![image](https://user-images.githubusercontent.com/10593890/89440198-27725e80-d700-11ea-9d4b-fec066d72f3b.png)